problems/[22.12.22] 1. Two Sum.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * https://leetcode.com/problems/two-sum/
+ * @param {number[]} nums
+ * @param {number} target
+ * @return {number[]}
+ */
+var twoSum = function (nums, target) {
+  const map = new Map();
+
+  for (let i = 0; i < nums.length; i++) {
+    const rest = target - nums[i];
+
+    if (map.has(nums[i])) return [i, map.get(nums[i])];
+    map.set(rest, i);
+  }
+};
```

Comment:
```suggestion
  }
  
  return [];
};
```
합이 `target`이 되는 경우가 없을 경우, 빈 배열을 반환해도 좋을 것 같습니다!